### PR TITLE
fix(FEV-1433): Ensure the visible text label for a control is included in the control's accessible name

### PR DESCRIPTION
**Pull request by lianbenjamin**
change "Done" button's text to "Close"

Solves [FEV-1433](https://kaltura.atlassian.net/browse/FEV-1433)

[FEV-1433]: https://kaltura.atlassian.net/browse/FEV-1433?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: translations/en.i18n.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
       "quiz_score": "Your score is {{quizScore}}",
       "retake_button": "Retake",
       "retake_button_area_label": "Click to retake the quiz",
-      "done_button": "Done",
-      "done_button_area_label": "Click to close the review",
+      "close_button": "Close",
```
[the remark this answers is not in the document]
Comment (lianbenjamin) [reply translations/en.i18n.json:28]: @SivanA-Kaltura there's a ticket for the translation

[the remark this answers is not in the document]
Comment (SivanA-Kaltura) [reply translations/en.i18n.json:28]: @lianbenjamin by the way, if the text already exists in the player you can reuse it. 
for example, the text for close exists here:
https://github.com/kaltura/playkit-js-ui/blob/master/translations/en.i18n.json#L51

so you can load it with something like withText({close: 'overlay.close'})

[the remark this answers is not in the document]
Comment (lianbenjamin) [reply translations/en.i18n.json:28]: @semarche-kaltura what do you think about using "overlay.close" from ui repo in ivq plugin?

[the remark this answers is not in the document]
Comment (semarche-kaltura) [reply translations/en.i18n.json:28]: > @semarche-kaltura what do you think about using "overlay.close" from ui repo in ivq plugin?

advantages: we don't need keep new translates (in our case duplicates with `overlay.close`).
disadvantages: if `overlay.close` got changed or if we need specific translate for `close_button` - we'll got the problem.
IMHO - let's keep translates related to IVQ inside IVQ repo

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]
PR looks good and if we have to deliver changes ASAP - go and merge.
I marked the PR as "Changes needed" only to got increased test coverage. If we're not in rush - cover the changes by e2e tests

**Review by semarche-kaltura — APPROVED** [review]

**Pull request merged**